src/meson.build

```diff
@@ -57,9 +57,9 @@ sources = files(
 	'string/i18n.c',
 	'string/xxd.c',
 
-	'../dependencies/nanopb/pb_common.c',
-	'../dependencies/nanopb/pb_encode.c',
-	'../dependencies/nanopb/pb_decode.c',
+	meson.current_source_dir() + '/../dependencies/nanopb/pb_common.c',
```

Comment:
Mmh, yeah, maybe it'll be better to move these types of libs under ./src, or make them full blown deps. Nanopb recommends compiling these files directly into the project, but I think I'd rather build a static lib.

For now, I'm fine with this coming in as-is.